app/Server/src/SERVERAPI/Views/Nutrients/FertilizerDetails.cshtml

```diff
@@ -189,7 +189,9 @@
                     </div>
                 </div>
                 <div style="clear:both"></div>
-                <hr style="height:1px;border:none;color:blue;background-color:lightblue;padding:0" />
+
+                <hr style="height:1px;border:none;color:blue;background-color:lightblue;padding:0; margin-top: 5px;" />
+
```

Comment:
Are changes to FertilizerDetails intended?

Reply:
Yes, I believe Luna had a good eye caught that on the wireframe and added the spacer 😎 , it's also in the description. 
![image](https://github.com/user-attachments/assets/6a8a5a6b-dfce-4f03-b180-7fd4897de282)